pdr_backend/lake/test/test_gql_data_factory.py

```diff
@@ -173,7 +173,7 @@ def test_get_gql_tables(mock_update):
 
     gql_dfs = gql_data_factory.get_gql_tables()
 
-    assert len(gql_dfs.items()) == 5
+    assert len(gql_dfs.items()) == 3
```

Comment:
This changed due to the 2 raw tables we stopped processing (slots and subscrptions)